dev-python/constantly/constantly-15.1.0~git.recipe

```diff
@@ -28,39 +28,49 @@ BUILD_REQUIRES="
 PYTHON_PACKAGES=(python39 python310)
 PYTHON_VERSIONS=(3.9 3.10)
 for i in "${!PYTHON_PACKAGES[@]}"; do
-pythonPackage=${PYTHON_PACKAGES[i]}
-pythonVersion=${PYTHON_VERSIONS[$i]}
-eval "PROVIDES_${pythonPackage}=\"\
-	${portName}_$pythonPackage = $portVersion\
-	\"; \
-REQUIRES_$pythonPackage=\"\
-	haiku\n\
-	cmd:python$pythonVersion\
-	\""
-BUILD_REQUIRES="$BUILD_REQUIRES
-	setuptools_$pythonPackage"
-BUILD_PREREQUIRES="$BUILD_PREREQUIRES
-	cmd:python$pythonVersion"
+	pythonPackage=${PYTHON_PACKAGES[i]}
+	pythonVersion=${PYTHON_VERSIONS[$i]}
+
+	eval "PROVIDES_${pythonPackage}=\"
+		${portName}_$pythonPackage = $portVersion
+		\""
+	eval "REQUIRES_$pythonPackage=\"
+		haiku
+		cmd:python$pythonVersion
+		\""
+	BUILD_REQUIRES+="
+		setuptools_$pythonPackage
+		"
+	BUILD_PREREQUIRES+="
+		cmd:python$pythonVersion
+		"
 done
 
 INSTALL()
 {
+	# Patch this call, otherwise we get:
+	# "setuptools.extern.packaging.version.InvalidVersion: [...]: 'refs-gh-pull-20-rebase'"
+	sed -i -e 's|version=versioneer.get_version()|version="15.1.0dev"|'  \
+		$sourceDir/setup.py
+
```

Comment:
This should probably be done in the `PATCH()` function (or maybe `BUILD()`). `INSTALL()` is a bit late for patching, especially because this is only done once and doesn't depend on the python version.